distributed/scheduler.py

```diff
@@ -2259,8 +2259,8 @@ def transition_processing_released(self, key: str, stimulus_id: str):
                 assert ts.state == "processing"
 
             w = _remove_from_processing(self, ts)
-            if w in self.workers:
-                worker_msgs[w] = [
+            if w:
+                worker_msgs[w.address] = [
```

Comment:
```suggestion
            ws = _remove_from_processing(self, ts)
            if ws:
                worker_msgs[ws.address] = [
```
Typically we'd name a WorkerState `ws`

Reply:
Good catch, thanks!